core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeDDLBuilder.scala

```diff
@@ -154,9 +155,23 @@ class SnowflakeDDLBuilder
     case c => ir.UnresolvedConstraint(c.getText)
   }
 
-  override def visitAlterSession(ctx: AlterSessionContext): ir.Catalog = {
+  override def visitAlterSession(ctx: AlterSessionContext): ir.UnresolvedCommand = {
     // Added replace formatting the incoming text a=b to a = b
-    ir.UnresolvedCatalog("ALTER SESSION SET " + ctx.sessionParams().getText.replace("=", " = "))
+    val alterSession = formatContextString(ctx)
+    ir.UnresolvedCommand(alterSession)
+  }
+
+  private def formatContextString(ctx: ParserRuleContext) = {
```

Comment:
Can we include it in Utils so that it can be reused?

Reply:
Done